securedrop/manage.py

```diff
@@ -22,13 +22,9 @@
 # We need to import config in each function because we're running the tests
 # directly, so it's important to set the environment correctly, depending on
 # development or testing, before importing config.
-#
-# TODO: do we need to store *_PIDFILE in the application config? It seems like
-# an implementation detail that is specifc to this management script.
-
 os.environ['SECUREDROP_ENV'] = 'dev'
 
-WORKER_PIDFILE = "/tmp/test_rqworker.pid"
+from config import WORKER_PIDFILE
```

Comment:
In general, this isn't great - I think the tests should handle the worker themselves to make it easier to control and modularize. This is part of the improvements that I have (currently just in my head) planned for 0.4.